Base/usr/share/man/man7/setuid_overview.md

```diff
@@ -0,0 +1,71 @@
+## Name
+
+Overview of real, effective, and saved user and group ids
+
+## Description
+
+Each process runs as user and a group. By default, a process started by a user has its user id set to that user and its group id set to that user's primary group.
+
+Each file has permissions that control if it can be read, written, and executed by its owner, other users in the owner's group, and by others. A process has the same access permissions as its user and its group.
+
+In addition to the access permissions bits, executables may have the "Set User ID"  and the "Set Group ID" bit set. Executables with these bits set run with the user or group id of the executable, instead of with the user and group id of the current user. These binaries are also called "SUID binaries" or "setuid binaries" (or "SGID binaries" or "setgid binaries" for the "Set Group ID" bit).
```

Comment:
Looks like GitHub just lost one of my comments; I'm not gonna retype the whole thing but the point was that "Executables with these bits set run with the user or group id of the executable" is not quite true, it just affects what happens at exec time.